packages/create-sitecore-jss/src/templates/nextjs-personalize/.env

```diff
@@ -12,3 +12,6 @@ NEXT_PUBLIC_CDP_SCRIPT_URL=https://d1mj578wat5n4o.cloudfront.net/boxever-1.4.8.m
 
 # Your Sitecore CDP point of sale
 NEXT_PUBLIC_CDP_POINTOFSALE=
+
+# Timeout for CDP requests to respond within (ms)
+PERSONALIZE_MIDDLEWARE_TIMEOUT=500
```

Comment:
Did we decide the default value to be 500?
I remember in some situations it took us more than 1000ms.

Reply:
set dev value for 10k now